src/OrbitGl/ImGuiTimeGraphLayout.h

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_GL_IM_GUI_TIME_GRAPH_LAYOUT_H_
+#define ORBIT_GL_IM_GUI_TIME_GRAPH_LAYOUT_H_
+
+#include <stdint.h>
+
+#include <algorithm>
+#include <cmath>
+
+#include "TimeGraphLayout.h"
+
+namespace orbit_gl {
+class ImGuiTimeGraphLayout : public TimeGraphLayout {
```

Comment:
A class name will help to understand why this class exists

Reply:
Do you have a suggestion for a different name?